Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1

```diff
@@ -61,7 +61,8 @@ $personalAccessToken = $vssEndpoint.Authorization.Parameters.AccessToken
 
 if ( [string]::IsNullOrEmpty($personalAccessToken))
 {
-  throw (Get-LocalizedString -Key "Unable to generate Personal Access Token for the user. Contact Project Collection Administrator")
+    Write-Host "##vso[task.logissue type=error;code=001002;]"
```

Comment:
repeated string throughout ... should we have a method for this, and errorType and code sent as params ?